src/main.rs

```diff
@@ -183,6 +183,42 @@ fn main() {
                                .value_name("LOG")
                                .takes_value(true))
 
+                          .arg(Arg::with_name("excl-line")
+                               .help("Lines in covered files containing this marker will be excluded.")
+                               .long("excl-line")
+                               .value_name("regex")
+                               .takes_value(true))
+
+                            .arg(Arg::with_name("excl-start")
+                                .help("Marks the beginning of an excluded section. The current line is part of this section.")
+                                .long("excl-start")
+                                .value_name("regex")
+                                .takes_value(true))
+
+                            .arg(Arg::with_name("excl-stop")
+                                .help("Marks the end of an excluded section. The current line is part of this section.")
+                                .long("excl-stop")
+                                .value_name("regex")
+                                .takes_value(true))
+
+                          .arg(Arg::with_name("excl-br-line")
+                               .help("Lines in covered files containing this marker will be excluded.")
+                               .long("excl-br-line")
+                               .value_name("regex")
+                               .takes_value(true))
+
+                            .arg(Arg::with_name("excl-br-start")
+                                .help("Marks the beginning of an excluded section. The current line is part of this section.")
+                                .long("excl-br-start")
+                                .value_name("regex")
+                                .takes_value(true))
+
+                            .arg(Arg::with_name("excl-br-stop")
+                                .help("Marks the end of an excluded section. The current line is part of this section.")
```

Comment:
... `.help("Marks the end of a section which is excluded from branch coverage. The current line not part of this section.")`